component/vshn_appcat_services.jsonnet

```diff
@@ -192,7 +192,7 @@ local vshn_appcat_service(name, serviceParams) =
      [if isOpenshift && std.objectHas(serviceParams, 'openshiftTemplate') then '21_openshift_template_%s_vshn' % name]: osTemplate,
 
    } else {})
-  + if vars.isSingleOrServiceCluster then {
+  + if vars.isSingleOrServiceCluster && inv.parameters.facts.cloud != 'exoscale' then {
```

Comment:
Can you add this into vars? like we do for `vars.isSingleOrServiceCluster`, something like `vars.isExoscale` / `vars.isCloudscale`